### DISCUSSION TITLE
simple-scan does not launch any window

### DISCUSSION THREAD

**Discussion by HyperCed** [Homebrew/discussions#2426] [Casks]
Hi,
I installed simple-scan when I was running MacOS 11 (Big Sur) and it worked great.
I upgraded MacOS to Monterey version 12 and now simple-scan does not launch any window...
There is no error but we cannot see any UI except the menu.

I tried brew update / upgrade and it seemed to work, but I still cannot see any window.

Is there something changing with gdk or something like that with Monterey ?

Thanks !

**Comment by cho-m** [comment]
Users have been reporting issues with various GTK-based formulae in combination with macOS Monterey like:
- `qalculate-gtk` - #2424
- `gedit` - Homebrew/homebrew-core#88743

There are some `gtk+3` upstream issues like https://gitlab.gnome.org/GNOME/gtk/-/issues/4395, https://gitlab.gnome.org/GNOME/gtk/-/issues/4342, ...

**EDIT:** Also may want to see if above issue or any other issue in upstream https://gitlab.gnome.org/GNOME/gtk/-/issues looks similar.

Further analysis is best done by a GTK developer/contributor.
